tests/integration/tier0/backup-release/main.fmf

```diff
@@ -1,6 +1,26 @@
-summary: variant help message
+summary: Backup os_release
+description: TODO
```

Comment:
Put something meaningful here please

Reply:
Good catch, thank you.
Description provided.